try {

} catch(${0:/* exception */}) {

}
